tools/ci_connector_ops/ci_connector_ops/qa_engine/cloud_availability_updater.py

```diff
@@ -8,28 +8,43 @@
 import subprocess
 import tempfile
 from pathlib import Path
-from typing import Iterable, Optional
+from typing import Iterable, List, Optional
 
 import git
 import requests
 
 from .constants import (
     AIRBYTE_PLATFORM_INTERNAL_GITHUB_REPO_URL,
+    AIRBYTE_PLATFORM_INTERNAL_ISSUES_ENDPOINT,
     AIRBYTE_PLATFORM_INTERNAL_MAIN_BRANCH_NAME,
     AIRBYTE_PLATFORM_INTERNAL_PR_ENDPOINT,
     AIRBYTE_PLATFORM_INTERNAL_REPO_OWNER,
+    GIT_USER_EMAIL,
+    GIT_USERNAME,
+    GIT_USERNAME_FOR_AUTH,
     GITHUB_API_COMMON_HEADERS,
+    GITHUB_API_TOKEN,
+    PR_LABELS,
 )
 from .models import ConnectorQAReport
 
 logger = logging.getLogger(__name__)
 
 
+def set_git_identity(repo: git.repo) -> git.repo:
+    repo.git.config("--global", "user.email", GIT_USER_EMAIL)
+    repo.git.config("--global", "user.name", GIT_USERNAME)
+    return repo
+
+
+def get_authenticated_repo_url(git_username: str, github_api_token: str) -> str:
+    return AIRBYTE_PLATFORM_INTERNAL_GITHUB_REPO_URL.replace("https://", f"https://{git_username}:{github_api_token}@")
```

Comment:
Was wondering why this was working for other scripts with https instead of git username/token, but it isn't used anywhere else 👍🏻